processor/transform/readme-to-mdx.ts

```diff
@@ -92,6 +92,30 @@ const readmeToMdx = (): Transform => tree => {
     }
   });
 
+  const hasExtraLinkAttrs = (attributes: MdxJsxAttribute[]) =>
+  !!attributes.find(attr => !['href', 'label'].includes(attr.name));
+
+  visit(tree, 'link', (link, index, parent) => {
+    if (!('data' in link)) return;
+    const attrs = {...link.data.hProperties};
+
+    if ('url' in link) attrs.href = link.url;
+    if ('url' in attrs) {
+      attrs.href = attrs.url;
+      delete attrs.url;
+    }
```

Comment:
Oops, my previous suggestion should then also change this part:

```suggestion
    const attrs = {...link};

    if ('url' in attrs) {
      attrs.href = attrs.url;
      delete attrs.url;
    }
```